draft-ietf-ppm-dap.md

```diff
@@ -3077,60 +3284,67 @@ where `batch_interval` is the batch interval requested by the Collector.
 
 ### Batch Buckets {#time-interval-batch-buckets}
 
-Each batch bucket is identified by an interval of time whose beginning is a
-multiple of the task's `time_precision` and whose duration is equal to the
+Each batch bucket is identified by an `Interval` whose duration is equal to the
 task's `time_precision`. The identifier associated with a given report is the
-unique such interval containing the timestamp of the report; for example, if
+unique such interval containing the timestamp of the report. For example, if
 the task's `time_precision` is 1000 seconds and the report's timestamp is
-1729629081, the relevant batch bucket identifier is `(1729629000, 1000)`. The
-first element of the pair denotes the start of the batch interval and the
-second denotes the duration.
-
-The `Query` received by the Leader determines a valid set of batch bucket
-identifiers if the following conditions hold (likewise for the `BatchSelector`
-received by the Helper):
+1729629081, the relevant batch bucket identifier is
 
-* `batch_interval.duration >= time_precision` (`time_precision` determines the
-  minimum batch duration)
+~~~ tls-presentation
+struct {
+  start = 1729629000,
+  duration = 1000,
+} Interval
+~~~
 
-* both `batch_interval.start` and `batch_interval.duration` are divisible by
-  `time_precision`
+The `Query` received by the Leader or `BatchSelector` received by the Helper
+determines a valid set of batch bucket identifiers if
+`batch_interval.duration >= time_precision`.
```

Comment:
Is `batch_interval` defined?

Reply:
No. I rewrote this in prose without `batch_interval` and I think it's clear.